### ISSUE TITLE
Add a Search button

### ISSUE THREAD

**Issue by aditigupta17** [OpenGenus/quark#31]
**This is a(n):**

- [x ] New feature proposal
- [ ] Reporting a keyword
- [ ] Error
- [ ] Proposal to the Search Engine
- [ ] Other

**Details:**

Currently the search engine gets triggered only after pressing the Enter key, it is not compatible for the users using the application on phones or any other touch-screen device per se. Hence, a Search button can be deployed next to the search textbox.

**Comment by palakjadwani** [comment]
Yes we can a search button alongside the search bar. I can work on it if needed.

**Comment by aditigupta17** [comment]
I have taken this up.
